### DISCUSSION TITLE
Deploy explorer in development

### DISCUSSION THREAD

**Discussion by daewookoh** [WalletConnect/walletconnect-monorepo#2579] [Developer Support]
I'm working with my flutter app

I want to test if redirction works fine 
with the inputs in explorer.

But I have to get reviewed
how can i provide Testing Instructions

**Comment by finessevanes** [comment]
Go to explorer.walletconnect.com. Select your project. Click on the "Explorer" tab. You can add your testing instructions here: 
<img width="783" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/44731696/04424dc2-9f09-440f-8179-896726cd8958">
